src/components/ui/ConditionalSelect.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { darkerprimary, palesecondary } from '../colors'
+
+export default class ConditionalSelect extends Component {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
+    onChange: PropTypes.func.isRequired,
+    options: PropTypes.object,
+    displayValue: PropTypes.string.isRequired,
+    subset: PropTypes.string
+  }
+
+  render() {
+    const {
+      displayValue,
+      label,
+      name,
+      onChange,
+      options,
+      required,
+      subset
+    } = this.props
+
+    return (
+      <label htmlFor={name}>
```

Comment:
Ich glaube, wenn das Label das Input ummantelt, brauchst du das for Attribut nicht.

Reply:
Oh, super, danke! Das räum ich dann nächste Woche direkt mal bei allen inputs auf.